src/vis/api.py

```diff
@@ -0,0 +1,83 @@
+from typing import List
+from datetime import date
+
+from ninja import Router, Query
+from ninja.security import django_auth
+from ninja.pagination import paginate
+from django.db import IntegrityError
+from django.views.decorators.csrf import csrf_exempt
+
+from registry.pagination import PagesPagination
+from main.schema import UnprocessableContentSchema, ForbiddenSchema
+from users.auth import UidKeyAuth
+from .models import ResultsProbLSTM, GeoMacroSaude
+from .schema import ResultsProbLSTMSchema, ResultsProbLSTMFilterSchema
+
+
+router = Router()
+uidkey = UidKeyAuth()
+
+
+@router.get(
+    "/vis/brasil/results-prob-lstm/",
+    response={
+        200: List[ResultsProbLSTMSchema],
```

Comment:
For now it is ok, But using a specific model type (LSTM) for the API endpoint is not  a good Idea.